src/PrestaShopBundle/Controller/Admin/Sell/Catalog/CategoryController.php

```diff
@@ -831,6 +846,16 @@ private function getErrorMessages()
                 $this->trans('An error occurred while uploading the image:', 'Admin.Catalog.Notification'),
                 $this->trans('You cannot upload more files', 'Admin.Notifications.Error')
             ),
+            UploadedImageConstraintException::class => [
+                UploadedImageConstraintException::EXCEEDED_SIZE => $this->trans(
+                    'Max file size allowed is "%d" bytes.', 'Admin.Notifications.Error',
```

Comment:
```suggestion
                    'Max file size allowed is "%s" bytes.', 'Admin.Notifications.Error',
```

Reply:
@Julievrz No, the file size is an integer not a string.

Reply:
@Progi1984 Thanks.  I guess the existing string was wrong too then. 
![image](https://user-images.githubusercontent.com/70583503/113157720-734bb800-923b-11eb-9564-11035e42f689.png)

Reply:
Approved ;)